pkgs/development/php-packages/imagick/default.nix

```diff
@@ -3,21 +3,8 @@
 buildPecl {
   pname = "imagick";
 
-  version = "3.4.4";
-  sha256 = "0xvhaqny1v796ywx83w7jyjyd0nrxkxf34w9zi8qc8aw8qbammcd";
-
-  patches = [
-    # Fix compatibility with PHP 8.
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Removed